dex/msgjson/types.go

```diff
@@ -869,6 +869,9 @@ type OrderBook struct {
 	Orders       []*BookOrderNote `json:"orders"`
 	BaseFeeRate  uint64           `json:"baseFeeRate"`
 	QuoteFeeRate uint64           `json:"quoteFeeRate"`
+	// RecentMatches is [rate, qty, timestamp]. Quantity is signed.
+	// Negative means that the maker was a sell order.
```

Comment:
Should this be **taker** was a sell order? Or does `Reset` have it switched?

Reply:
Tested and taker sells loaded this way show as red (correct), so looks like the comment is just a typo.